DQMOffline/Trigger/python/HTMonitor_cff.py

```diff
@@ -0,0 +1,232 @@
+
+# HLT_MET200
+MET200_HTmonitoring = hltHTmonitoring.clone()
+MET200_HTmonitoring.FolderName = cms.string('HLT/HT/MET200/')
+MET200_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_MET200_v*")
+
+# HLT_MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_IDTight
+MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_HTmonitoring = hltHTmonitoring.clone()
+MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_HTmonitoring.FolderName = cms.string('HLT/HT/MonoCentralPFJet80_PFMETNoMu110/')
+MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_IDTight_v*")
+MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_HTmonitoring.jetSelection      = cms.string("pt[0] > 100 && eta[0] < 2.5 && neutralHadronEnergyFraction[0] < 0.8 && chargedHadronEnergyFraction[0] > 0.1")
+MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_HTmonitoring.muoSelection      = cms.string("isTightMuon()")
+MonoCentralPFJet80_PFMETNoMu110_PFMHTNoMu110_HTmonitoring.jetSelection_HT   = cms.string("pt > 30 && eta < 2.5")
+
+# HLT_MonoCentralPFJet80_PFMETNoMu120_PFMHTNoMu120_IDTight
+MonoCentralPFJet80_PFMETNoMu120_PFMHTNoMu120_HTmonitoring = hltHTmonitoring.clone()
+MonoCentralPFJet80_PFMETNoMu120_PFMHTNoMu120_HTmonitoring.FolderName = cms.string('HLT/HT/MonoCentralPFJet80_PFMETNoMu120/')
+MonoCentralPFJet80_PFMETNoMu120_PFMHTNoMu120_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_MonoCentralPFJet80_PFMETNoMu120_PFMHTNoMu120_IDTight_v*")
+MonoCentralPFJet80_PFMETNoMu120_PFMHTNoMu120_HTmonitoring.jetSelection_HT   = cms.string("pt > 30 && eta < 2.5")
+
+# HLT_MonoCentralPFJet80_PFMETNoMu130_PFMHTNoMu130_IDTight
+MonoCentralPFJet80_PFMETNoMu130_PFMHTNoMu130_HTmonitoring = hltHTmonitoring.clone()
+MonoCentralPFJet80_PFMETNoMu130_PFMHTNoMu130_HTmonitoring.FolderName = cms.string('HLT/HT/MonoCentralPFJet80_PFMETNoMu130/')
+MonoCentralPFJet80_PFMETNoMu130_PFMHTNoMu130_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("MonoCentralPFJet80_PFMETNoMu130_PFMHTNoMu130_IDTight_v*")
+MonoCentralPFJet80_PFMETNoMu130_PFMHTNoMu130_HTmonitoring.jetSelection      = cms.string("pt[0] > 100 && eta[0] < 2.5 && neutralHadronEnergyFraction[0] < 0.8 && chargedHadronEnergyFraction[0] > 0.1")
+MonoCentralPFJet80_PFMETNoMu130_PFMHTNoMu130_HTmonitoring.muoSelection      = cms.string("isTightMuon()")
+MonoCentralPFJet80_PFMETNoMu130_PFMHTNoMu130_HTmonitoring.jetSelection_HT   = cms.string("pt > 30 && eta < 2.5")
+
+# HLT_MonoCentralPFJet80_PFMETNoMu140_PFMHTNoMu140_IDTight
+MonoCentralPFJet80_PFMETNoMu140_PFMHTNoMu140_HTmonitoring = hltHTmonitoring.clone()
+MonoCentralPFJet80_PFMETNoMu140_PFMHTNoMu140_HTmonitoring.FolderName = cms.string('HLT/HT/MonoCentralPFJet80_PFMETNoMu140/')
+MonoCentralPFJet80_PFMETNoMu140_PFMHTNoMu140_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("_v*")
```

Comment:
please fix the path name to `HLT_MonoCentralPFJet80_PFMETNoMu140_PFMHTNoMu140_IDTight_v*`